wayland-display-core/src/comp/input.rs

```diff
@@ -192,29 +193,29 @@ impl State {
                 self.last_pointer_movement = Instant::now();
                 let horizontal_amount = event
                     .amount(Axis::Horizontal)
-                    .or_else(|| event.amount_discrete(Axis::Horizontal).map(|x| x * 2.0))
+                    .or_else(|| event.amount_v120(Axis::Horizontal).map(|x| x * 2.0))
```

Comment:
```suggestion
                    .or_else(|| event.amount_v120(Axis::Horizontal).map(|x| x * 2.0 / 120.0))
```

The new v120 are different.